src/ui/Base/ComponentOptions.ts

```diff
@@ -687,6 +687,11 @@ export class ComponentOptions {
         template = ComponentOptions.loadResultTemplateFromId(id);
       }
     }
+
+    if (template == null) {
+      template = ComponentOptions.loadResultTemplateFromPageTemplatesCache();
```

Comment:
This create a `TemplateList`, with everything that's registered in `pageTemplate`, whenever it fails to load a template when an option is built using `buildTemplateOption` in component options.

I guess this *can* work for ResultList, but makes no sense for all the other component that uses this option. Like `Quickview`, `Matrix`, `Folding`, etc.

A `TemplateList` is a special class that is designed to iterate over multiple result template, and conditionally load the first one that match a condition.

This is not needed for `Quickview`, for example.

Shouldn't this be done specifically in the `ResultList` itself ? When it tries to load it's template option ?